incl/footer.html

```diff
@@ -12,6 +12,7 @@
     <div id="block-aggregator-feed-1" class="clear-block block block-aggregator">
       <h2 class="title">Recent news</h2>
       <div class="content">
+        <p><strong>Please participate in:</strong> the <a href="https://sondages.inria.fr/index.php/356388?lang=en">Coq community survey 2022</a> (by 28 February 2022, AoE). <a href="https://coq.discourse.group/t/coq-community-survey-2022/1547/2">More details</a>.</p>
```

Comment:
```suggestion
        <p><strong>Please participate in the <a href="https://sondages.inria.fr/index.php/356388?lang=en">Coq community survey 2022</a></strong> (by 28 February 2022, AoE). <a href="https://coq.discourse.group/t/coq-community-survey-2022/1547/2">More details</a>.</p>
```
If we want to use bold, I think it makes more sense to have the whole thing in bold.

Reply:
This is what I had done initially but I thought it didn't look good in addition to the link style.

Reply:
In this case, maybe better to not have any boldness at all.

Reply:
Without any boldness, it seems to me that it's really hard to notice that something has changed in the side panel (when you're used to ignoring it).

Reply:
I think the best indicator would be to put a box around the text. Maybe with a pale blue background even to really catch the eye.

Reply:
Indeed, that's a good idea. I've found a color that is even better suited than light blue given the overall website style:

![image](https://user-images.githubusercontent.com/1108325/152939746-8c5c5278-9c71-4b46-b06d-978b1a34930d.png)

Reply:
I think it is almost perfect. Are you able to tweak the padding on the left so that it doesn't touch the start of those sentences? But if not, don't worry.

Reply:
Here we go.